cpp/tests/test_odesecir.cpp

```diff
@@ -1177,5 +1178,206 @@ TEST(Secir, read_population_data_one_age_group)
+            .finished();
+
+    ASSERT_THAT(print_wrap(model1[0].populations.array().cast<double>()),
+                MatrixNear(print_wrap(model2[0].populations.array().cast<double>()), 1e-5, 1e-5));
+
+    ASSERT_THAT(print_wrap(model1[0].populations.array().cast<double>()),
+                MatrixNear(print_wrap(model3[0].populations.array().cast<double>()), 1e-5, 1e-5));
+
+    ASSERT_THAT(print_wrap(model1[0].populations.array().cast<double>()),
+                MatrixNear(print_wrap(expected_values), 1e-5, 1e-5));
+    ASSERT_THAT(print_wrap(model2[0].populations.array().cast<double>()),
+                MatrixNear(print_wrap(expected_values), 1e-5, 1e-5));
+    ASSERT_THAT(print_wrap(model3[0].populations.array().cast<double>()),
+                MatrixNear(print_wrap(expected_values), 1e-5, 1e-5));
+}
+
+TEST(TestOdeSecir, export_time_series_init)
+{
+    TempFileRegister temp_file_register;
+    auto tmp_results_dir = temp_file_register.get_unique_path();
+    ASSERT_THAT(mio::create_directory(tmp_results_dir), IsSuccess());
+
+    constexpr auto num_age_groups = 6; //reading data requires RKI data age groups
+    mio::osecir::Model<double> model(make_model(num_age_groups));
+
+    // Test exporting time series
+    std::vector<mio::osecir::Model<double>> models{model};
+    ASSERT_THAT(mio::osecir::export_input_data_county_timeseries(
+                    models, tmp_results_dir, {1002}, {2020, 12, 01}, std::vector<double>(size_t(num_age_groups), 1.0),
+                    1.0, 2, mio::path_join(TEST_DATA_DIR, "county_divi_ma7.json"),
+                    mio::path_join(TEST_DATA_DIR, "cases_all_county_age_ma7.json"),
+                    mio::path_join(TEST_DATA_DIR, "county_current_population.json")),
+                IsSuccess());
+
+    auto data_extrapolated = mio::read_result(mio::path_join(tmp_results_dir, "Results_rki.h5"));
+    ASSERT_THAT(data_extrapolated, IsSuccess());
+
+    // Values were generated by the tested function export_input_data_county_timeseries;
+    // can only check stability of the implementation, not correctness
+    auto expected_results =
+        mio::read_result(mio::path_join(TEST_DATA_DIR, "export_time_series_initialization_osecir.h5")).value();
+
+    ASSERT_THAT(print_wrap(data_extrapolated.value()[0].get_groups().matrix()),
+                MatrixNear(print_wrap(expected_results[0].get_groups().matrix()), 1e-5, 1e-5));
+}
+
+TEST(TestOdeSecir, export_time_series_init_old_date)
+{
+    TempFileRegister temp_file_register;
+    auto tmp_results_dir = temp_file_register.get_unique_path();
+    ASSERT_THAT(mio::create_directory(tmp_results_dir), IsSuccess());
+
+    constexpr auto num_age_groups = 6; //reading data requires RKI data age groups
+    mio::osecir::Model<double> model(make_model(num_age_groups));
+
+    // Test exporting time series
+    std::vector<mio::osecir::Model<double>> models{model};
+    ASSERT_THAT(mio::osecir::export_input_data_county_timeseries(
```

Comment:
As we expect a log_warn here, please change the log level to keep the output clean.

Reply:
good point :), thank you